packages/nextjs/components/scaffold-eth/Contract/WriteOnlyFunctionForm.tsx

```diff
@@ -87,7 +89,11 @@ export const WriteOnlyFunctionForm = ({
   });
 
   return (
-    <div className="flex flex-col gap-3">
+    <div
+      className={`flex gap-3 ${
+        inputs.length === 0 && !functionFragment.payable ? "flex-row justify-between items-center" : "flex-col"
+      } ${index === 0 ? "pt-1 pb-5" : "py-5"}`}
```

Comment:
Same here, we can use the `first:` css selector.

Reply:
🙌 fixed at [6c57dcd](https://github.com/scaffold-eth/se-2/pull/223/commits/6c57dcdb817ebddefdee227a4309d5298d517590)